src/include/sof/schedule/ll_schedule.h

```diff
@@ -34,7 +34,7 @@ struct ll_task_pdata {
 	uint16_t skip_cnt;	/**< how many times the task was skipped for execution */
 };
 
-#ifndef __ZEPHYR__
+#if !defined(__ZEPHYR__) || defined(CONFIG_IMX)
```

Comment:
It's still not easy to test this without knowing which `--board` argument to pass to `west` (see #4217) but at least I now see exactly what you're doing and I can simulate it, thx.

Reply:
I've explained how I tested it with both `xtensa-build-zephyr.sh`  and `west `commands: see [here](https://github.com/thesofproject/sof/pull/4525#issuecomment-884742627)
The only problem is that the pull request from Zephyr's repo is not yet merged, so you'll need to use a specific branch: https://github.com/iuliana-prodan/zephyr/commits/imx_sof_with_zephy